sound/soc/bcm/rpi-simple-soundcard.c

```diff
@@ -3,6 +3,7 @@
  * rpi-simple-soundcard.c -- ALSA SoC Raspberry Pi soundcard.
  *
  * Copyright (C) 2018 Raspberry Pi.
+ * Copyright (C) 2019 Jon Ronen-Drori <jon_ronen@yahoo.com>
```

Comment:
It's slightly poor form to claim copyright for such a small contribution to a file, IMHO.

Reply:
I'll remove this

Reply:
Removed